joplin/locations/tests.py

```diff
@@ -1,3 +1,24 @@
+from django.core.exceptions import ValidationError
 from django.test import TestCase
+from django.core.exceptions import NON_FIELD_ERRORS
+import pdb
+from locations.models import LocationPageRelatedServices
+from . import models, factories
+from contextlib import contextmanager
 
-# Create your tests here.
+
+class LocationPageRelatedServiceTests(TestCase):
```

Comment:
Hey a test! This tests to see if the validation error works correctly

Reply:
@Niicck if you're setting up tests you might want to see if this one gets picked up along the way?